### PR TITLE
test: add liquid assets script to test network

### PR DESCRIPTION
**Pull request by bee344**
Two questions I have regarding this:
* Currently I have to force pjs to accept `<any>` when dealing with the `asset-conversion-pallet`, because the types it expects come from the `substrate-node` implementation and are incompatible with the `Kusama Asset Hub` implementation, but due to this I have to disable `eslint` for those two lines.
```typescript
const createLiquidityPoolCall = (api: ApiPromise) => {
	// For now, we have to override the types of the Assets until PJS is updated
	// eslint-disable-next-line @typescript-eslint/no-unsafe-argument
	return api.tx.assetConversion.createPool(<any>native, <any>asset);
};

const addLiquidityCall = (api: ApiPromise, amountNative: number, amountAsset: number, to: KeyringPair) => {
	// For now, we have to override the types of the Assets until PJS is updated
	// eslint-disable-next-line @typescript-eslint/no-unsafe-argument
	return api.tx.assetConversion.addLiquidity(<any>native, <any>asset, amountNative, amountAsset, 0, 0, to.address);
};
```
* Trappist doesn't support `asset-conversion` so all the process runs on KAH, wondering how we feel about that.

**Edit**: [Issue](https://github.com/paritytech/trappist/issues/281) to add asset-conversion to trappist.
**Edit 2**: [Types](https://github.com/polkadot-js/api/blob/7a8a4f1c64ce5dbb5bf56558054c7e906e81d133/packages/types-augment/src/lookup/types-substrate.ts#L5229) on PJS that are used on Asset Conversion

### CODE REVIEW

File: scripts/testNetworkLiquidAssets.ts [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+};
+
+const mintCall = (api: ApiPromise, to: KeyringPair, amount: number) => {
+	return api.tx.assets.mint(ASSET_ID, to.address, amount);
+};
+
+const createLiquidityPoolCall = (api: ApiPromise) => {
+	// For now, we have to override the types of the Assets until PJS is updated
+	// eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+	return api.tx.assetConversion.createPool(<any>native, <any>asset);
```
Comment (TarikGul) [inline scripts/testNetworkLiquidAssets.ts:47]: This is totally fine with me. I double checked the ts compilter issue happening with PJs and this is the right approach.

File: scripts/testNetworkLiquidAssets.ts [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+};
+
+const addLiquidityCall = (api: ApiPromise, amountNative: number, amountAsset: number, to: KeyringPair) => {
+	// For now, we have to override the types of the Assets until PJS is updated
+	// eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+	return api.tx.assetConversion.addLiquidity(<any>native, <any>asset, amountNative, amountAsset, 0, 0, to.address);
```
Comment (TarikGul) [inline scripts/testNetworkLiquidAssets.ts:53]: Same with this, totally fine.

**Comment by TarikGul** [comment]
> Trappist doesn't support asset-conversion so all the process runs on KAH, wondering how we feel about that.

I think this should be totally fine, not much we can do. That being said if we can still create the liquid pool assets thats a step forward. In this case though we just can't transfer them to Trappist if I understand this correctly?

**Review by TarikGul — APPROVED** [review]
Overall, amazing script. LGTM!

**Comment by bee344** [comment]
> > Trappist doesn't support asset-conversion so all the process runs on KAH, wondering how we feel about that.
> 
> I think this should be totally fine, not much we can do. That being said if we can still create the liquid pool assets thats a step forward. In this case though we just can't transfer them to Trappist if I understand this correctly?

Correct, right now there's no way to send the LP tokens to Trappist, but we create them just fine and we can even send them between accounts, so that's ok.

**Comment by TarikGul** [comment]
> > > Trappist doesn't support asset-conversion so all the process runs on KAH, wondering how we feel about that.
> > 
> > 
> > I think this should be totally fine, not much we can do. That being said if we can still create the liquid pool assets thats a step forward. In this case though we just can't transfer them to Trappist if I understand this correctly?
> 
> Correct, right now there's no way to send the LP tokens to Trappist, but we create them just fine and we can even send them between accounts, so that's ok.

Okay I think that's perfect then and the script does exactly what it intends on doing. Feel free to merge it in, whenever your'e ready

**Pull request merged**